omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/summary/RegistrationSummaryFragmentController.java

```diff
@@ -48,12 +50,19 @@ public void controller(FragmentConfiguration config,
             app = appFrameworkService.getApp((String) config.get("appId"));
         }
         model.addAttribute("appId", app !=null ? app.getId() : "referenceapplication.registrationapp.registerPatient");
-
-        List<Extension> registrationFragments = appFrameworkService.getExtensionsForCurrentUser("registrationSummary.contentFragments", appContextModel);
-        Collections.sort(registrationFragments);
-        model.addAttribute("registrationFragments", registrationFragments);
-
+       
+        List<Extension> firstColumnFragments = appFrameworkService.getExtensionsForCurrentUser("registrationSummary.contentFragments", appContextModel);
         List<Extension> secondColumnFragments = appFrameworkService.getExtensionsForCurrentUser("registrationSummary.secondColumnContentFragments", appContextModel);
+        
+        if (CollectionUtils.isEmpty(firstColumnFragments) || CollectionUtils.isEmpty(secondColumnFragments)) {
+        	List<Extension> extensions = RegistrationSummaryExtensionsGenerator.generate(app, true);
```

Comment:
Unfortunately, as per the ticket's description, the default should be:
```java

RegistrationSummaryExtensionsGenerator.generate(app, false)
```

Reply:
I had done the former just to have a feel this on an Openmrs Instance. Noted 👍 .